src/components/modeler/Selection.vue

```diff
@@ -585,6 +585,7 @@ export default {
       this.updateSelectionBox();
       if (this.isMultiplayer) {
         window.ProcessMaker.EventBus.$emit('multiplayer-updateNodes', this.getProperties(this.selected));
+        window.ProcessMaker.EventBus.$emit('multiplayer-updateNodes', this.getConectedLinkProperties(this.conectedLinks));
```

Comment:
```suggestion
        window.ProcessMaker.EventBus.$emit('multiplayer-updateNodes', this.getConnectedLinkProperties(this.connectedLinks));
```

Reply:
change the variable not mentioned here from `this.conectedLinks` to `this.connectedLinks`